applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaBoxROI.cpp

```diff
@@ -19,7 +19,7 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#include "CudaTypes.h"
+#include <sofa/gpu/cuda/CudaTypes.h>
```

Comment:
Same SofaCuda/type/CudaTypes

might be a big mess!

Reply:
if we want really to be consistent with SOFA, it is SofaCuda/defaulttype/CudaTypes (like sofa/defaulltype/VecTypes, etc)